.github/minty.yaml

```diff
@@ -4,7 +4,8 @@ rule:
   if: |-
     assertion.iss == issuers.github &&
     assertion.repository_owner_id == '93787867' &&
-    assertion.repository_id == '576289489'
+    assertion.repository_id == '576289489' ||
+    assertion.iss == issuers.google && assertion.sub == '112892967544648233217'
```

Comment:
Do we really want to allow _anything_ with that subject to mint a token? What is that integer?

Reply:
That subject is the unique ID for the integration test project's WIF service account. Unfortunately GCP-minted tokens don't seem to include a lot of metadata to work with.

Reply:
We could also check that the audience points to this repo?

Reply:
I think this is the best we can do, according to `https://accounts.google.com/.well-known/openid-configuration` I dont see any values to use to identify the caller other than the service account number.

Reply:
Could we at least add a comment inline

Reply:
We could use email? Service accounts have their number built into the email right?

Reply:
nvm, thats project number

Reply:
Email would be better. Emails don't have ServiceAccountIds; they only have project numbers.

Reply:
Should we worry about squatting, seems project number would mitigate that?

Reply:
Will change this to use email, but first I need to put out a PR to make token minter parse the `email` claim from tokens. 

Service account emails have project IDs but not project numbers, e.g. `test-dmearl-sa@dmearl-gma-test-82.iam.gserviceaccount.com`. Project IDs can eventually be reused, which could be a security risk. What do we think?

Reply:
Sticking with SA number for now, but the email claim is exposed now so we can switch to that easily if we want to

Reply:
Project IDs cannot be reused